src/brasil/gov/portlets/portlets/audio.py

```diff
@@ -84,7 +85,7 @@ def audio(self):
         portal_state = getMultiAdapter((self.context, self.request),
                                        name=u'plone_portal_state')
         portal = portal_state.portal()
-        if isinstance(audio_path, unicode):
+        if isinstance(audio_path, str):
```

Comment:
I think this doesn't make sense: IIRC a `''`string in Python 2 is equivalent to a `b''` string in Python 3; you have to check the whole code block; check this:

http://python-future.org/compatible_idioms.html#basestring

@loechel could you please give us some advise on writing future-proof code here?

Reply:
besides that the code looks buggy: if a path has to be a `basestring` why in the world we are returning it as `unicode` in the first place? that's has to be fixed on a different way.

Reply:
:+1: